lesson002/task5/t5.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+        <link rel="stylesheet" href="css/maine.css">
+
+        <title>Site</title>
+    </head>
+    <body class="bg-dark">
+        <div class="container">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+            <a class="navbar-brand" href="t5.html"><i>MySite</i></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navAlt" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navAlt">
+              <div class="navbar-nav">
+                <a class="nav-link active" href="projects.html">My projects<span class="sr-only">(current)</span></a>
+                <a class="nav-link" href="contacts.html">Contacts</a>
+              </div>
+            </div>
+          </nav>
+        </div>
+        <div class="container">
+          <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+            <div class="carousel-inner card-img-bottom">
+              <div class="carousel-item active">
+                <img src="img/1500x500.jfif" class="d-block w-100" alt="banner 1">
+                <div class="carousel-caption">
+                  <p class="lead">Hello, My name is Anastasia.<p class="lead">This is my first site!</p></p>
+                </div>
+              </div>
+              <div class="carousel-item">
+                <img src="img/1500x500 (1).jfif" class="d-block w-100" alt="banner 2">
+                <div class="carousel-caption">
+                  <h3>GitHub</h3>
+                  <p class="lead">Please see my GitHub.</p>
+                  <a class="btn btn-large btn-outline-light" href="https://github.com/AnMatveeva">My GitHub</a>
+                </div>
+              </div>
+            </div>
+           <button class="carousel-control-prev bg-secondary border-0" type="button" data-target="#carouselExampleControls" data-slide="prev">
+              <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+              <span class="sr-only">Previous</span>
+            </button>
+            <button class="carousel-control-next bg-secondary border-0" type="button" data-target="#carouselExampleControls" data-slide="next">
+              <span class="carousel-control-next-icon" aria-hidden="true"></span>
+              <span class="sr-only">Next</span>
+            </button>
```

Comment:
Look at the carousel buttons. They step on the heder. Is that how it should be?

Reply:
Done